lib/pbench/test/unit/server/test_controllers_list.py

```diff
@@ -206,10 +223,7 @@ def test_empty_query(
     @pytest.mark.parametrize(
         "exceptions",
         (
-            {
-                "exception": requests.exceptions.HTTPError,
-                "status": HTTPStatus.BAD_GATEWAY,
-            },
+            {"exception": make_http_exception(502), "status": HTTPStatus.BAD_GATEWAY},
```

Comment:
Shouldn't that be `HTTPStatus.BAD_GATEWAY` instead of `502`?

Reply:
I hadn't updated all of the subclasses with names, and I lazily copied-and-pasted the calls from one where I hadn't. Sigh.